### DISCUSSION TITLE
Do you have develop plan for NXP S32Z/E?

### DISCUSSION THREAD

**Discussion by PeaceZhang** [bao-project/bao-hypervisor#48] [General]
REF: https://www.nxp.com/products/processors-and-microcontrollers/s32-automotive-platform/s32z-and-s32e-real-time-processors/s32z2-safe-and-secure-high-performance-real-time-processors:S32Z2?subscribe=true&

**Comment by sandro2pinto** [comment]
We are completing support for the S32S24 real-time processor, which is similar to the S32Z/E.

Feel free to reach out to info@bao-project.org.

**Reply by PeaceZhang** [reply]
Thank you for your reply, as of i know, NXP already decided not to take s32s to mass production. 

ref: https://community.nxp.com/t5/Blogs/The-Role-of-S32Z-E-Real-Time-Processors-in-the-Software-Defined/ba-p/1483895

if you add s32z/e to your support device list, maybe more people can find BAO project

**Reply by sandro2pinto** [reply]
Yes S32S24 was the available sample for initial development.
Mass production of S32Z/E is expected only for 2024, but the transition from S32S to S32Z/E should be very smooth.

It is indeed very clear to us that the more architectutes, CPUs, and platforms we support, the more likely we drive adoption.

That's why we also have support for the generic Armv8-R AArch32 / AArch64 for the Arm FVP - upstream soon.

Perhaps you can ping us (info@bao-project.org) and share a bit more about your interest / target / use case. 

Otherwise feel free to take the generic Armv8-R AArch32 support and add support for the S32Z/E and contribute back.

**Reply by PeaceZhang** [reply]
Now I am just investigate the feasibility for S32G/E be next generation of vehicle's zone controller and complete my  investigation report. thank you for your reply.

**Reply by sandro2pinto** [reply]
Anything you could share with us, we would appreciate. Thank you.
